src/config.rs

```diff
@@ -394,6 +394,7 @@ pub struct DbConfig {
     pub compaction_readahead_size: ReadableSize,
     pub info_log_max_size: ReadableSize,
     pub info_log_roll_time: ReadableDuration,
+    pub info_log_keep_log_file_num: u64,
```

Comment:
I think we can default to 10 * 1GB log files.

Reply:
ok